src/Apps/Settings/Routes/EditProfile/helpers/verifyID.ts

```diff
@@ -0,0 +1,36 @@
+import { verifyIDMutation } from "__generated__/verifyIDMutation.graphql"
+import { commitMutation, Environment, graphql } from "react-relay"
+
+export const verifyID = async (relayEnvironment: Environment) => {
```

Comment:
**suggestion:** Let's use hooks for mutations ([example](https://github.com/artsy/force/blob/2201cae43e7a07108f8f6895b4a41732a46d33d6/src/Apps/MyCollection/Routes/EditArtwork/Mutations/useCreateArtwork.ts#L5)). This is the preferred practice in Force and like this we don't have to pass the relay environment 🤓